packages/elements/src/photon-multirx-form/index.tsx

```diff
@@ -40,414 +42,481 @@ export type Address = {
 };
 
-customElement(
-  'photon-prescribe-workflow',
-  {
-    patientId: undefined,
-    templateIds: undefined,
-    templateOverrides: undefined,
-    prescriptionIds: undefined,
-    hideSubmit: false,
-    hideTemplates: true,
-    enableOrder: false,
-    enableLocalPickup: false,
-    enableSendToPatient: false,
-    enableMedHistory: false,
-    mailOrderIds: undefined,
-    pharmacyId: undefined,
-    loading: false,
-    address: undefined,
-    weight: undefined,
-    weightUnit: 'lbs',
-    triggerSubmit: false,
-    setTriggerSubmit: undefined,
-    toastBuffer: 0
-  },
-  (
-    props: {
-      patientId?: string;
-      templateIds?: string;
-      templateOverrides?: TemplateOverrides;
-      prescriptionIds?: string;
-      hideSubmit: boolean;
-      hideTemplates: boolean;
-      enableOrder: boolean;
-      enableLocalPickup: boolean;
-      enableSendToPatient: boolean;
-      enableMedHistory: boolean;
-      mailOrderIds?: string;
-      pharmacyId?: string;
-      loading: boolean;
-      address?: Address;
-      weight?: number;
-      weightUnit?: string;
-      triggerSubmit: boolean;
-      setTriggerSubmit?: (val: boolean) => void;
-      toastBuffer: number;
-    },
-    options
-  ) => {
-    let ref: any;
-    const { store, actions } = createFormStore({
-      dispenseAsWritten: false,
-      patient: undefined,
-      treatment: undefined,
-      draftPrescriptions: [],
-      pharmacy: undefined,
-      errors: [],
-      address: undefined
+type PrescribeProps = {
+  patientId?: string;
+  templateIds?: string;
+  templateOverrides?: TemplateOverrides;
+  prescriptionIds?: string;
+  hideSubmit: boolean;
+  hideTemplates: boolean;
+  enableOrder: boolean;
+  enableLocalPickup: boolean;
+  enableSendToPatient: boolean;
+  enableMedHistory: boolean;
+  enableCombineAndDuplicate: boolean;
+  mailOrderIds?: string;
+  pharmacyId?: string;
+  loading: boolean;
+  address?: Address;
+  weight?: number;
+  weightUnit?: string;
+  triggerSubmit: boolean;
+  setTriggerSubmit?: (val: boolean) => void;
+  toastBuffer: number;
+  formStore?: any;
+  formActions?: any;
+};
+
+function PrescribeWorkflow(props: PrescribeProps) {
+  let ref: any;
+
+  const client = usePhoton();
+  const [showForm, setShowForm] = createSignal<boolean>(
+    !props.templateIds && !props.prescriptionIds
+  );
+  const [errors, setErrors] = createSignal<FormError[]>([]);
+  const [isLoading, setIsLoading] = createSignal<boolean>(true);
+  const [isEditing, setIsEditing] = createSignal<boolean>(false);
+  const [authenticated, setAuthenticated] = createSignal<boolean>(
+    client?.authentication.state.isAuthenticated || false
+  );
+  const [, recentOrdersActions] = useRecentOrders();
+
+  onMount(() => {
+    if (props.address) {
+      // if manually overriding address, update the store on mount
+      props.formActions.updateFormValue({
+        key: 'address',
+        value: props.address
+      });
+    }
+    if (ref) {
```

Comment:
What happens if theres no ref?

Reply:
I'm not sure if it's even possible... onMount is triggered when the component is mounted so I think guaranteed to have a ref. this might be superfluous, I'm gonna remove it, throw caution to the wind